dashboard/app/entities.go

```diff
@@ -514,3 +517,64 @@ func timeDate(t time.Time) int {
 	year, month, day := t.Date()
 	return year*10000 + int(month)*100 + day
 }
+
+type RecipientType int
+
+const (
+	To RecipientType = iota
+)
+
+func (t RecipientType) String() string {
+	return [...]string{"To", "Cc"}[t]
+}
+
+type RecipientInfo struct {
+	Address mail.Address
+	Type    RecipientType
+}
+
+type Recipients []RecipientInfo
+
+func (r Recipients) Len() int           { return len(r) }
+func (r Recipients) Less(i, j int) bool { return r[i].Address.Address < r[j].Address.Address }
+func (r Recipients) Swap(i, j int)      { r[i], r[j] = r[j], r[i] }
+
+func (r Recipients) String() string {
+	return strings.Join(r.StringSlice(), "|")
+}
+
+func (r Recipients) StringSlice() []string {
+	slice := []string{}
+	for _, addr := range r {
+		slice = append(slice, addr.Address.Address+"\n"+addr.Type.String())
```

Comment:
Using Type.String() here is fragile.

Consider, this is merged, somebody later sees this:
```
func (t RecipientType) String() string {
	return [...]string{"To", "Cc"}[t]
}
```
No comments, nothing, looks like this is used for logging or something. They decide that "CC" looks better than "Cc". Do the change. Boom. We just ruined the production database.